app/uk/gov/hmrc/timetopayproxy/config/FeatureSwitch.scala

```diff
@@ -18,9 +18,4 @@ package uk.gov.hmrc.timetopayproxy.config
 
 import play.api.Configuration
 
-case class FeatureSwitch(value: Option[Configuration]) {
-
-  def isTTPDropTwo: Boolean =
-    value.flatMap(_.getOptional[Boolean]("ttpDropTwoEnabled")).getOrElse(false)
-
-}
+case class FeatureSwitch(value: Option[Configuration]) {}
```

Comment:
```suggestion
case class FeatureSwitch(value: Option[Configuration])
```
or
```suggestion
case class FeatureSwitch(value: Option[Configuration]) {
  // Implement feature switch getter methods here.
}
```

Reply:
I see, either of these changes makes it more obvious that either the case class is empty or needs to be filled out. I think I'll make the second change as that makes more sense with wanting to keep feature switches in.